### DISCUSSION TITLE
LPS measurement value

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1257] [Q&A]
Hi, I tried to use UWB to measure the reliability of the data. I set the distance to 100 centimeters, and then used VM to observe the measured value, and found that there was an error of 30 centimeters. Is this normal?
[IMG_5837.zip](https://github.com/bitcraze/discussions/files/14784392/IMG_5837.zip)
![VirtualBox_BitcrazeVM_28_03_2024_13_24_25](https://github.com/bitcraze/discussions/assets/118436188/e945bc2d-cdc8-4a8f-8474-bb829a2c95a2)

**Comment by knmcguire** [comment]
That's a bit much but not that ordinary. This can be due to multiple things:

- Reflective surfaces in the area
- Antenna lobs (best is to have the antenna's facing eachother directly
- an antenna delay that is off.

However, the idea of the EKF is that these biases should be able to cancel eachother out usually.
